packages/esm-ward-app/src/config-schema.ts

```diff
@@ -79,9 +73,51 @@ export const configSchema: ConfigSchema = {
         },
       },
     },
-    identifierElementDefinitions: {
+    pendingItems: {
+      _type: Type.Array,
+      _description: 'Configures pending orders and transfers to display.',
+      _default: [
+        {
+          id: 'pending-items',
+          orders: {
+            orderTypes: [{ label: 'Labs', uuid: '52a447d3-a64a-11e3-9aeb-50e549534c5e' }],
+          },
+          showPendingItems: true,
+        },
+      ],
+      _elements: {
+        id: {
+          _type: Type.String,
+          _description: 'The unique identifier for this patient card element',
+        },
+        orders: {
+          orderTypes: {
+            _type: Type.Array,
+            _description: 'Configures pending orders and transfers to display.',
+            _elements: {
+              uuid: {
+                _type: Type.UUID,
+                _description: 'Identifies the order type.',
+              },
+              label: {
+                _type: Type.String,
+                _description:
+                  "The label or i18n key to the translated label to display. If not provided, defaults to 'Orders'",
+                _default: null,
+              },
+            },
+          },
+        },
+        showPendingItems: {
+          _type: Type.Boolean,
+          _description:
+            'Optional. If true, pending items (e.g., number of pending orders) will be displayed on the patient card.',
+        },
+      },
+    },
+    patientIdentifier: {
       _type: Type.Array,
-      _description: `Defines patient identifier elements that can be included in the card header or footer. The default element 'patient-identifier' displays the preferred identifier.`,
+      _description: `Configures patient identifier to display. An unconfigured element displays the preferred identifier.`,
       _default: [
```

Comment:
looks like the `_default` should be an empty array, provided the default object passed is missing some keys i.e `identifierTypeUuid`